src/wallet/api/wallet.h

```diff
@@ -166,10 +166,16 @@ class WalletImpl : public Wallet
                                         std::set<uint32_t> subaddr_indices = {}) override;
     virtual PendingTransaction * createSweepUnmixableTransaction() override;
     bool submitTransaction(const std::string &fileName) override;
+    bool submitTransactionFromString(const std::string &fileName) override;
     virtual UnsignedTransaction * loadUnsignedTx(const std::string &unsigned_filename) override;
+    virtual UnsignedTransaction * loadUnsignedTxFromString(const std::string &unsigned_filename) override;
```

Comment:
Keyword `virtual` should be used in the interface file `wallet2_api.h` only. (I know this is violated already in several places in the current API.)

And parameter name `unsigned_filename` should be `data`.